src/plugins/saved_objects_management/public/management_section/objects_table/components/import_summary.scss

```diff
@@ -0,0 +1,24 @@
+.savedObjectsManagementImportSummary__row {
+  margin-bottom: $euiSizeXS;
+}
+
+.savedObjectsManagementImportSummary__title {
+  // Constrains title to the flex item, and allows for truncation when necessary
+  min-width: 0;
+}
+
+.savedObjectsManagementImportSummary__createdCount {
+  color: $euiColorSuccess;
```

Comment:
You'll need to be sure to use the text version of all these colors when applying them to text, ie
```suggestion
  color: $euiColorSuccessText;
```

Reply:
Oops, will do, thanks!